config/katello-answers.yaml

```diff
@@ -34,11 +34,13 @@ foreman::plugin::hooks: false
 foreman::plugin::tasks: true
 foreman::plugin::chef: false
 foreman::plugin::default_hostgroup: false
+foreman::plugin::openscap: false
 foreman::plugin::puppetdb: false
 foreman::plugin::setup: false
 foreman::plugin::templates: false
 foreman::plugin::remote_execution: false
 
+foreman_proxy::plugin::openscap: false
```

Comment:
I think the answers file shouldn't be altered directly, the migration handles that as far as I know -- @mbacovsky ?